packages/flutter_markdown/CHANGELOG.md

```diff
@@ -156,198 +160,198 @@
 
 ## 0.6.10
 
- * Update `markdown` dependency
```

Comment:
Can you revert the format change? It makes this PR noisy?

Reply:
@domesticmouse I had noticed the reformat had accidentally slipped through and I was going to revert it when I added the test however I realised it had only reformatted the second half of the file (0.6.10 and earlier) as the first half of the changelog was already formatted according to best practices. So it made the changelogs second half consistent with the first which I thought maybe wasn't a bad thing?

Happy to revert it though if you think keeping the pr tidy is better?  Let me know and I'll update it the same day. Thanks

Reply:
I'll defer to @stuartmorgan on this one